core/src/main/java/cz/xtf/core/bm/TestCaseContext.java

```diff
@@ -0,0 +1,28 @@
+package cz.xtf.core.bm;
+
+/**
+ * Used only in case when test cases are executed in different namespaces when:
+ * -Dxtf.openshift.namespace.per.testcase=true
+ */
+public class TestCaseContext {
+
+    /**
+     *
+     * This allows to track currently running test case for correct namespace mapping. This is used to automatically find
+     * namespace for running test case when
+     * creating {@link cz.xtf.core.openshift.OpenShift} instances.
+     */
+    private static String testCaseName;
```

Comment:
field and getter/setter does not match

testCaseName vs. runningTestCaseName

Reply:
Right, fixed.